src/app/static/src/js/components/runWorkflow/runWorkflowCreate.vue

```diff
@@ -1,44 +1,182 @@
 <template>
-    <div>
+    <div id="create-workflow-container">
         <h2 id="create-workflow-header">Run workflow</h2>
-        <div class="pt-2">
+        <div class="pt-2 col-sm-6">
             <p>Either:</p>
-            <button id="create-workflow" @click="create()" type="button" class="btn btn-success">
+            <button id="create-workflow" @click="create()"
+                    type="button" class="btn btn-success">
                 Create a blank workflow
             </button>
         </div>
-        <div id="report-list" class="pt-4">
+        <div id="report-list" class="pt-4 col-sm-8">
             <p>Or re-use an existing workflow:</p>
-            <!-- use vue-select -->
+            <div id="v-select">
+                <v-select label="name"
+                          :filter="searchWorkflows"
+                          :options="workflows.slice(0, 10)"
+                          v-model="selectedWorkflow"
+                          placeholder="Search by name or user...">
+                    <template id="optionTemplate" #option="{ name, email, date }">
```

Comment:
There wasn't another (easy) option here, but reminds me that we should consider how we refer to users in both workflow and report-running. We're variously using name, username and email address at the moment. When it comes to searching name is probably more intuitive than email address, but we seem to be just displaying first name elsewhere in the reporting portal. Something for another ticket.